extensions/wikia/CreateNewPageFlow/CreateNewPageFlow.setup.php

```diff
@@ -0,0 +1,4 @@
+<?php
+
```

Comment:
Please add some information what this extension is about. Also, I suggest to rename it to FlowTracking to match the rest of direction we assumed